cypress.json

```diff
@@ -9,5 +9,7 @@
   "videosFolder": "client/cypress/videos",
   "supportFile": "client/cypress/support/index.js",
   "defaultCommandTimeout": 20000,
-  "requestTimeout": 15000
+  "requestTimeout": 15000,
+  "viewportWidth": 1280,
```

Comment:
@gabrieldutra Look: I changed default viewport dimensions for Cypress, and I expected to see a lot of diffs, but see only few ones (https://percy.io/Redash/redash/builds/2137256). Does that mean that pages that we already captured were properly resized by CSS? (sorry if it's a dumb question, I'm just trying to not miss anything)

Reply:
> Does that mean that pages that we already captured were properly resized by CSS? 

Exactly, the only pages that showed a diff are ones that rely on JavaScript to determine size, the others should not change as in Percy the width is the same.

Reply:
> ...regarding the two other visualizations that rely on JS...

I hope so :thinking:

Reply:
Upd: yes, seems they where scaled using vertical dimensions (because their height > width), and since container has fixed height they're unchanged. The first one has width > height and therefore was scaled using horizontal dimensions which depend on viewport size.